### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by BrahimDahmani**
I'm newbie on gems contribution, this is my first attempt.

I would like to write some tests, but i can't run existent tests, if you tell me how to run them, maybe i could help on writing somes.

ps: sorry for bad english

### CODE REVIEW

File: lib/pager_api/pagination/kaminari.rb [hunk 1]
```diff
@@ -13,8 +13,7 @@ def paginate(*args)
         options[:meta] = meta(paginated_collection, options) if PagerApi.include_pagination_on_meta?
 
         pagination_headers(paginated_collection) if PagerApi.include_pagination_headers?
-
-        render options
+        render json: options
```
Comment (kurenn) [inline lib/pager_api/pagination/kaminari.rb]: Why are you adding the json key?

**Comment by kurenn** [comment]
There is currently no specs, you are very welcome to set it up!

**Comment by kurenn** [comment]
There is an attempt on this pull request https://github.com/IcaliaLabs/pager-api/pull/2/files

File: lib/pager_api/pagination/kaminari.rb (hunk 1 above)
Comment (BrahimDahmani) [inline lib/pager_api/pagination/kaminari.rb]: there is a problem with `render options`, to solve that we could use `return options` and in the controller we can use it like this

```ruby
books = paginate Book.all
render json: books
```

or we could use `render json: options` and 
```ruby
books = Book.all
paginate books
```
as mentioned in the doc

so i chose the second solution to keep the the gem api intact

Comment (kurenn) [inline lib/pager_api/pagination/kaminari.rb]: I'm not sure if this would work, are testing this on Rails 5?

Comment (BrahimDahmani) [inline lib/pager_api/pagination/kaminari.rb]: yes  i tested it

**Pull request merged**